_data/projects/.net-compiler-platform-roslyn.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: .NET Compiler Platform ("Roslyn")
+desc: The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic
+  compilers with rich code analysis APIs.
+site: https://github.com/dotnet/roslyn
+tags:
+- roslyn
+- csx
+- script
+- compiler
+- C#
+- VB.NET
```

Comment:
this doesn't match the syntax as described in the readme (i.e. `tags: [roslyn, csx, ...]`)?